push_notification_ui/static/js/utils.js

```diff
@@ -0,0 +1,12 @@
+
+export function getServerPublicKey() {
+    return 'BJeeceDtmhorvlG5v9annBsWIMlBXAU1f6nWoIZ0uxFZIA4jVkoeshPwgZ1B_GBI4g2fnfhuJCw48qSzZPNBDOI'
```

Comment:
I thought you created an API for this? This works fine to though in this project.